docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: "2.2"
+version: "3.8"
```

Comment:
Tää versionumero on nykyään deprekoitu ja suositellaan vaan jättämään se pois: https://docs.docker.com/compose/compose-file/04-version-and-name/#version-top-level-element

Reply:
jees poistin